tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -349,7 +349,7 @@ export class RunsTableContainer implements OnInit, OnDestroy {
   addHparamFilter(event: FilterAddedEvent) {
     this.store.dispatch(
       hparamsActions.dashboardHparamFilterAdded({
-        name: event.header.name,
```

Comment:
On inspection, it appears filter actions can be independent of "Headers". We just use the name field of the headers anyway.

Reply:
I probably would have made a quick PR with just this as a standalone change. This is fine though.

Reply:
Ack - I'll split these sorts of changes going forward